src/Components/Home/Users.js

```diff
@@ -80,8 +80,8 @@ function Users({ list }) {
                 className="m-2 w-16rem px-3 py-2 transition-all transition-duration-300"
                 template={
                   <>
-                    <Avatar
-                      image={user.avatar}
+                    <Chip
```

Comment:
Great work so far. I think we can go a little further in order to simplify this as much as possible and get rid of any unused code so would you be able to update the template to just this:

`template={
                  <span className="text-overflow-ellipsis white-space-nowrap overflow-hidden">{user.name}</span>
                }`

Also, it's fine to delete the unused imports Avatar, and also Badge and Utils once the template is updated as that keeps the file as clean as possible and reduces tech debt. Let us know if you have any questions.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix